src/app/components/named-entity/named-entity.component.html

```diff
@@ -16,35 +16,34 @@
         {{ entity.label }}
         <evt-pinner [item]="entity" renderer="NamedEntity" pinType="NamedEntities"></evt-pinner>
     </div>
-    <div class="card-body ne-content" *ngIf="contentOpened">
-        <!-- Tab panes -->
-        <ngb-tabset [justify]="false">
-            <ngb-tab [title]="'Info' | translate">
-                <ng-template ngbTabContent>
+    <div class="card-body ne-content d-none" [class.d-block]="contentOpened">
+        <ngb-tabset [justify]="false" #entityDetails="ngbTabset">
+            <ngb-tab id="info" [title]="'Info' | translate" [disabled]="entity.content.length === 0">
+                <ng-template ngbTabContent *ngIf="contentOpened">
```

Comment:
careful with `*ngIf` in `ng-template`!
https://angular.io/guide/structural-directives